backend/routes/auth.py

```
@@ -0,0 +1,276 @@
+"""
+Authentication routes for the admin dashboard.
+
+Provides session-based authentication using secure cookies with proper admin user management.
+"""
+
+import os
+
+# Import the proper admin authentication system
+import sys
+from typing import Dict, Optional
```

Ruff (0.12.2)

`typing.Optional` imported but unused

Remove unused import: `typing.Optional`

(F401)